model/pipeline/src/main/proto/org/apache/beam/model/pipeline/v1/beam_runner_api.proto

```diff
@@ -1329,7 +1332,7 @@ message Trigger {
   message AfterSynchronizedProcessingTime {
   }
 
-  // The default trigger. Equivalent to Repeat { AfterEndOfWindow } but
+  // The default trigger. Equivalent to AfterEndOfWindow { Late: Always }} but
```

Comment:
These are equivalent, but I do prefer the phrasing you have altered it to.

Reply:
AFAICT They are not.

AfterEndOfWindow is defined as having the sub triggers as implicitly repeating with no default behavior if a sub trigger isn't defined (technically, it defaults to the never trigger). So, Repeat { AfterEndOfWindow } has only an ontime, and maybe a closing firing. The repeat isn't semantically useful.